### ISSUE TITLE
BCC went missing, what can I do?

### ISSUE THREAD

**Issue by wujie234** [fyookball/electrum#98]
Hi Fyookbal ,

Thank you for prompt response to my issue below. So if you commented this issue normally happens when the user is connected to the wrong server. Whose server do you mean? What can I do now? Want to get the BCC back... 
Can you please give me some advice? Shall I wait? Shall I contact someone or some team? Thanks a lot!
--------------------------------------------------------------------------------------------------------------------
2nd time: BCC showed up then went missing...

Paperwallet via blockchain.info send all fund to Mycelium
Mycelium send btc to Ledger
Mycelium seed to restore in electroncash (HERE STARTED ISSUE)
firstly the BCC fund didn't showup in electroncash account immediately
I waited several hours (went sleep and get up) then I happily saw BCC showed up in electroncash but somehow showing "Not parenting" or "not patenting" in red. I thought it might be the network slow issue. So I went sleep again.
I waited one more day, log in again... the 2nd time try BCC disappeared... It only shows the 1st time testing history. The 2nd time "Not parenting"BCC history two lines are even disappeared.
I am scared now... Can any one help me with that? Thanks a lot!!!

**Comment by fyookball** [comment]
bottom right corner of the client there is a green or orange or red circle

**Comment by wujie234** [comment]
Thank you. I tried. But the the BCC is not showing... is it possible when this "unconfirmed parent" happened, it sent back to somewhere? where it possibly be. I sent BTC from Mycelium to Ledger, then restore Mycelium's seeds to electroncash. If re-sent, where can it possibly be sent to?

**Comment by fyookball** [comment]
check on block explorer.

**Comment by wujie234** [comment]
I tried to switch the block explorer, however nothing changed... :(
My question now:
Mycelium seed to restore in electroncash (HERE STARTED ISSUE)
it showed "not parenting" with the correct amount... two transaction lines... if it got sent back, where are the private keys now? Thank yoo!!!
----------------------------------------------------------------------

Paperwallet via blockchain.info send all fund to Mycelium
Mycelium send btc to Ledger
Mycelium seed to restore in electroncash (HERE STARTED ISSUE)
firstly the BCC fund didn't showup in electroncash account immediately
I waited several hours (went sleep and get up) then I happily saw BCC showed up in electroncash but somehow showing "Not parenting" or "not patenting" in red. I thought it might be the network slow issue. So I went sleep again.
I waited one more day, log in again... the 2nd time try BCC disappeared... It only shows the 1st time testing history. The 2nd time "Not parenting"BCC history two lines are even disappeared.
